packages/core/src/command/consolidated/consolidated-delete.ts

```diff
@@ -20,6 +21,20 @@ export async function deleteConsolidated({
 }: DeleteConsolidatedCommand): Promise<void> {
   const { log } = out(`deleteConsolidated - cx ${cxId}, pat ${patientId}`);
 
+  await Promise.all([
+    deleteConsolidatedFromS3({ cxId, patientId }),
+    deleteConsolidatedFromSearchEngine({ cxId, id: patientId }),
```

Comment:
We should not delete from search engine, otherwise data won't be available until the consolidated bundle is recreated and reingested. Reingestion takes care of removing old data, so let's not do this here.